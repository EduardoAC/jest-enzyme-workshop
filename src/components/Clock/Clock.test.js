import React from 'react'
import { shallow } from 'enzyme'
import { Clock } from './Clock'


describe('<clock />', () => {
  /**
   * What test cases you recommended for this clock component?
   */
  it('renders', () => {
    const wrapper = shallow(<Clock />)
    expect(wrapper).toMatchSnapshot()
  })
})