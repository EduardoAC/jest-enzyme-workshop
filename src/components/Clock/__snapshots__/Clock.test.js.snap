// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<clock /> renders 1`] = `
<p>
  12:29:46
</p>
`;
